.github/workflows/main.yml

```diff
@@ -88,5 +88,8 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/ci-setup
 
+      - name: build
+        run: yarn build
```

Comment:
Would I be correct in assuming this is added because the tsconfigs used for dtslint don't use a module resolution mode that supports the imports/exports fields so the build is necessary so preconstruct outputs stuff that for the default case doesn't require supporting the imports/exports fields?

Reply:
seems like the answer is yes from reading the commits